target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c

```diff
@@ -1540,9 +1520,8 @@ static int ag71xx_probe(struct platform_device *pdev)
 		goto err_phy_disconnect;
 	}
 
-	pr_info("%s: Atheros AG71xx at 0x%08lx, irq %d, mode:%s\n",
-		dev->name, (unsigned long) ag->mac_base, dev->irq,
-		ag71xx_get_phy_if_mode_name(ag->phy_if_mode));
```

Comment:
if you want, ag71xx_get_phy_if_mode_name() could also be replaced by: [phy_modes()](https://elixir.bootlin.com/linux/v4.14.96/source/include/linux/phy.h#L113)

The phy mode string will be in lower case, but that should be the only downside.

Reply:
That's pretty good. I've changed to use that.